contrastsecurity/README.md

```diff
@@ -0,0 +1,44 @@
+## Contrast Security - Protect ScreenBoard for DataDog
+
+### Set up Contrast Protect logs collection
+
+You need to enable logs collection for Datadog Agent in `~/.datadog-agent/datadog.yaml` on other paltforms, please find the location of your config file [here](https://docs.datadoghq.com/agent/guide/agent-configuration-files/?tab=agentv6):
+```
+logs_enabled: true
+```
+
+* Create a new folder `java.d` in the `conf.d/` directory of DataDog configuration directory.
+* Create a new conf.yaml file.
+* Add a custom log collection configuration group.
+```
+logs:
+  - type: file
+    path: /path/to/contrast/security.log
+    service: contrast
+    source: java
+```
+
+* Restart DataDog Agent.
+
+More info: https://docs.datadoghq.com/logs/log_collection/?tab=tailexistingfiles#getting-started-with-the-agent
+
+### Create a Logs Processing Pipeline
+
+* Create a new pipeline in DataDog dashboard (Logs -> Configuration - > New Pipeline) 
+* Expand the new pipeline and click Add Processor
+* Add a Grok Parser to the pipeline with the following parsing rule
+```
+ContrastSecurityLogRule %{data:data}pri=%{data:pri} src=%{ip:src} spt=%{number:spt} request=%{data:request} requestMethod=%{word:requestMethod} app=%{data:app} outcome=%{word:outcome}
+```
+
+### Import Contrast Dashboard into DataDog
```

Comment:
```suggestion
### Import contrast dashboard into Datadog
```

Reply:
Hi. Contrast Security is the name of the company. So could we keep Contrast capitalized

Reply:
Ah, gotcha. Then let's just change it to `Import Contrast dashboard into Datadog`. Note that `Datadog` is not camel cased.